build/jshint/xfail.js

```diff
@@ -1,34 +1,20 @@
-// custom jshint reporter, allows errors in files listed in the
```

Comment:
I'm wondering if we still need this reporter ? Maybe to summarize the output, but maybe jshint has standard reporters for this already ? 

At the very least we should maybe rename it.